statvar_imports/ndap/nss_education/pvmap/Average expenditure per student during current academic year pursuing technical or professional course_metadata.csv

```diff
@@ -0,0 +1,16 @@
+parameter,value
+url,https://ndap.niti.gov.in/dataset/7271?tab=data

```

Comment:
pls don't use spaces in the filename.
Pls rename to a shorter filename, say, education_expenditure_professional_course_{metadata,pvmap}.csv